src/lib/y2storage/planned/bcache.rb

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage/planned/device"
+require "y2storage/planned/mixins"
+require "y2storage/match_volume_spec"
+
+module Y2Storage
+  module Planned
+    # Specification for a Y2Storage::Bcache object to be created during the proposal
+    #
+    # @see Device
+    class Bcache < Device
+      include Planned::CanBeFormatted
+      include Planned::CanBeMounted
+      include Planned::CanBeEncrypted
+      include MatchVolumeSpec
+
+      # @return [name] device name of the MD RAID
```

Comment:
First of all, ~~MD RAID~~~ bcache device.

On the other hand, is it realistic to expect the device actually gets this name? If not, I would describe this as the tentative name, at most.